libs/core/algorithms/include/hpx/parallel/algorithms/detail/find.hpp

```diff
@@ -34,8 +34,7 @@ namespace hpx::parallel::detail {
             sequential_find_t<ExPolicy>, Iterator first, Sentinel last,
             T const& value, Proj proj = Proj())
         {
-            return util::loop_pred<
-                std::decay_t<hpx::execution::sequenced_policy>>(
+            return util::loop_pred<ExPolicy>(
```

Comment:
Could you add `static_assert` verifying that `ExPolicy` is actually a sequential policy?

Reply:
Change had been made so sequential_find_t can accept sequential and unsequential execution policies. Do you want me to rather use `static_assert(is_seq || is_unseq)` ?